build_runner/lib/src/generate/build_impl.dart

```diff
@@ -465,7 +465,7 @@ class BuildImpl {
     }
 
     var inputSets = packages.map((package) => new InputSet(
-        package, [package == _packageGraph.root.name ? '**/*' : 'lib/**']));
+        package, [package == _packageGraph.root.name ? '**' : 'lib/**']));
```

Comment:
Seems like a pre-existing issue, but this looks like its collecting a lot of things we don't actually want (such as hidden dirs like `.git` etc). Technically you could consume these I guess, but it seems like a bad idea. Not sure the best way to get around this though

I filed https://github.com/dart-lang/build/issues/200 to investigate this, as I don't think this change makes it any worse :).